controls/pcidss_4.yml

```diff
@@ -2045,1274 +2629,1420 @@ controls:
         - service_auditd_enabled
```

Comment:
- `grub2_audit_argument` is already in `10.7.2`.
  - The `10.2.1` is more about enabling the auditing and I believe `10.7.2` is a better candidate for these specific configurations.
- `grub2_audit_backlog_limit_argument` included in 10.7.2. Thanks for finding it.
- `rsyslog_cron_logging`, `rsyslog_logging_configured`, `service_systemd-journald_enabled`, `service_rsyslog_enabled`were included to related rules.
  - The PCI-DSS is not specific about products and solutions so I prefer to include less than more rules.
  - But keeping them in related_rules is good to keep us aware and we can reconsider this along the time.

Changes in 11a80be3c3a55afd9e0e4c5df0a7ecde5e49a3f1

Reply:
I do not think that rules which make processes auditable before the audit daemon starts belong to 10.7.2, I see them more as assurance that the logging is active as soon as possible... but this can be disputed later, I don't think it is something what should block this PR.

Reply:
Sure @vojtapolasek . We can better discuss this in a separate PR. I believe along the time we will find many other opportunities of improvements to polish the control file. Thanks for the hints and all valid points you shared during your reviews.